dev/com.ibm.ws.transport.http/src/com/ibm/ws/genericbnf/internal/BNFHeadersImpl.java

```diff
@@ -4497,6 +4525,69 @@ private void processForwardedAddressExtract(String nodeExtract, ListType type) {
         list.add(nodeName);
     }
 
+    /*
+     * A valid proto may start with an alpha followed by any number of chars that are
+     * - alpha
+     * - numeric
+     * - "+" or "-" or "."
+     */
+
+    private boolean validateProto(String forwardedProto) {
```

Comment:
These two new methods look like they would have benefited from being unit tested? Suggest that you open a follow up PR to add that if possible.